src/kernel/context.h

```diff
@@ -19,7 +19,7 @@ namespace kernel {
 //! should be stored to std::unique_ptr members pointing to opaque types.
 struct Context {
```

Comment:
Some of the option fields do currently hold references and it might be a good idea to move ownership of those referred values to the kernel Context eventually. I agree though that the comment should reflect the current code, not some future hypothetical.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/29252#discussion_r1593012869

> Some of the option fields do currently hold references and it might be a good idea to move ownership of those referred values to the kernel Context eventually.

Fair enough, new members could be added and kernel::Context could become something more like what was originally described, and the new comment I suggested would need to get updated in that case. To be sure, I don't think any changes need to be made as part of this PR. I was more making an observation on the current state of things. It seems worth noting out that kernel context object plays a pretty different role than node and wallet context objects because its job is initialize external state, while node and wallet context objects are meant to be passive containers.